internal/model/manifest.go

```diff
@@ -102,6 +103,14 @@ func (m Manifest) validate() *ValidateErr {
 	return nil
 }
 
+func (m1 Manifest) Equal(m2 Manifest) bool {
+	primitivesMatch := m1.Name == m2.Name && m1.K8sYaml == m2.K8sYaml && m1.DockerRef == m2.DockerRef && m1.BaseDockerfile == m2.BaseDockerfile && m1.StaticDockerfile == m2.StaticDockerfile && m1.StaticBuildPath == m2.StaticBuildPath
+	cmdMatch := m1.Entrypoint.Equal(m2.Entrypoint)
+	pmMatch := m1.FileFilter == m2.FileFilter
```

Comment:
i don't think this will work for any non-trivial file filter?

Reply:
also, did you forget to compare ConfigFiles?

Reply:
and Mounts

Reply:
These are in now.

Reply:
(after #493 is landed, this'll have to be changed a bit and then it should be good to merge)